apps/style/javalab/style.scss

```diff
@@ -11,4 +11,18 @@ body {
 
 .main {
   padding: 0;
-}
\ No newline at end of file
+}
+
+.cm-content, .cm-gutter { 
+  min-height: 390px;
+}
+
+.cm-gutters { margin: 1px; }
+
+.cm-scroller { overflow: auto; }
+
+.cm-wrap { border: 1px solid silver }
+
+#codeWorkspace {
+  border: none !important;
```

Comment:
In my shameless crusade to get rid of `!important` in our code base, I wanted to double check: Is `!important` truly important here?

Reply:
Yes, without `!important` there is a white border at the top of the page.